includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -331,6 +331,7 @@ public function setup_wizard_footer() {
 			<?php elseif ( 'recommended' === $this->step || 'activate' === $this->step ) : ?>
 				<a class="wc-setup-footer-links" href="<?php echo esc_url( $this->get_next_step_link() ); ?>"><?php esc_html_e( 'Skip this step', 'woocommerce' ); ?></a>
 			<?php endif; ?>
+			<?php do_action( 'wc_setup_footer' ); ?>
```

Comment:
All hooks should start with `woocommerce_`, we have a few named wrong with `wc_`, but we should consolidate this in the future.